src/design_docs/_users/lib/actions.coffee

```diff
@@ -0,0 +1,49 @@
+validate = require('./validation/validate')
+do_action = require('./shared/do_action')
+validate_doc_update = require('./shared/validate_doc_update').validate_doc_update
+h = require('./helpers')
+_ = require('./underscore')
+
+a = {}
+a.do_actions =
```

Comment:
For readability I might recommend leaving a comment here about what each letter action does (or wherever is most appropriate authoritative source including the README).